src/core/navigation.h

```diff
@@ -32,12 +32,16 @@ class Navigation : public QObject
 
     Q_PROPERTY( QgsPoint location READ location WRITE setLocation NOTIFY locationChanged )
     Q_PROPERTY( QgsPoint destination READ destination WRITE setDestination NOTIFY destinationChanged )
+    Q_PROPERTY( QString destinationName READ destinationName NOTIFY destinationNameChanged )
 
     Q_PROPERTY( QgsGeometry path READ path NOTIFY detailsChanged )
     Q_PROPERTY( double distance READ distance NOTIFY detailsChanged )
     Q_PROPERTY( QgsUnitTypes::DistanceUnit distanceUnits READ distanceUnits NOTIFY detailsChanged )
     Q_PROPERTY( double bearing READ bearing NOTIFY detailsChanged )
 
+    Q_PROPERTY( int destinationFeatureCurrentVertex READ destinationFeatureCurrentVertex NOTIFY destinationFeatureCurrentVertexChanged )
+    Q_PROPERTY( int destinationFeatureVertices READ destinationFeatureVertices NOTIFY destinationFeatureVerticesChanged )
```

Comment:
Can we call that `destinationFeatureVertexCount`, I know it's longer but I think it's the QGIS "standard", "vertices" would indicate a QList